iXBRLViewerPlugin/iXBRLViewer.py

```diff
@@ -132,23 +131,6 @@ def escapeJSONForScriptTag(self, s):
         """
         return s.replace("<","\\u003C").replace(">","\\u003E").replace("&","\\u0026")
 
-    def makeLanguageName(self, langCode):
-        code = re.sub("-.*","",langCode)
-        try:
-            language = pycountry.languages.lookup(code)
```

Comment:
`pycountry` is no longer used. Can you drop the import and remove the dependency from `pyproject.toml`?